administrator/language/en-GB/com_joomlaupdate.ini

```
@@ -188,6 +188,7 @@ COM_JOOMLAUPDATE_XML_DESCRIPTION="Updates Joomla to the latest version with one
 ;Copy of INSTL constants (Pre-Update check)
 INSTL_DATABASE_SUPPORT="Database Support:"
 INSTL_DATABASE_SUPPORTED="Database Supported (%s)"
+INSTL_DATABASE_MIN_VERSION="Database Version (%s => %s)"
```

```suggestion
INSTL_DATABASE_MIN_VERSION="Database Version (%s >= %s)"
```
Why not use `>=` (greater or equal)?

---

I copied it from PHP text string

---

ah, no